infrastructure/static/css/webflow.css

```diff
@@ -1026,10 +1023,10 @@ textarea.w-select {
   -ms-user-select: none;
   user-select: none;
 }
-.w-slider-arrow-left [class^="w-icon-"],
-.w-slider-arrow-right [class^="w-icon-"],
-.w-slider-arrow-left [class*=" w-icon-"],
-.w-slider-arrow-right [class*=" w-icon-"] {
+.w-slider-arrow-left [class^='w-icon-'],
+.w-slider-arrow-right [class^='w-icon-'],
+.w-slider-arrow-left [class*=' w-icon-'],
```

Comment:
Does the leading space matter? `class*=' w-icon-'`

Reply:
This should be okay. It's generated by Webflow and we don't know of any issues. Will take it up with Matt though, even if just to tidy up the CCS.

I also pulled master in and that sorted out the download button ID